MyApplication/app/build.gradle

```diff
@@ -25,12 +27,17 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
 
-    compile 'com.android.support:appcompat-v7:24.2.0'
```

Comment:
Gradle should be divided to testing part and regular/compile/run part. 
Please create sections.